mailpoet/lib/Newsletter/Scheduler/AutomationEmailScheduler.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace MailPoet\Newsletter\Scheduler;
```

Comment:
I'm not sure, to me, it seems that this belongs to MailPoet "core", to the `Scheduler` namespace indeed. In `MailPoet\Automation\Integrations\MailPoet` we have only actions, triggers, subjects, and stuff like that.

The naming is not great but ultimately, "automation" should replace "automatic". It will take some time, of course. Maybe I'll just add the check as mentioned in https://github.com/mailpoet/mailpoet/pull/4227#discussion_r915163888?

Reply:
I agree that this does feel right to have in the core plugin, not sure what I was thinking last week.